package.json

```
@@ -104,6 +104,8 @@
     "test-deno": "deno run --allow-env --allow-read --allow-net --allow-run --allow-sys --allow-write ./test/deno.js",
     "test-rs": "START_REPLICA_SET=1 mocha --timeout 30000 --exit ./test/*.test.js",
     "test-tsd": "node ./test/types/check-types-filename && tsd",
+    "test-encryption": "mocha --exit ./test/encryption/*.test.js",
+    "test-encryption-local": "chmod +x scripts/run-encryption-tests-local.sh && scripts/run-encryption-tests-local.sh",
```

```suggestion
    "test-encryption-local": "bash scripts/run-encryption-tests-local.sh",
```

Usually, we'd just invoke the script with bash instead of chmod'ing it and invoking it directly

---

Sounds good, made the change!